tests/buildall/recipes/buildall.yaml

```diff
@@ -1,15 +1,17 @@
 inherit: ['basement::rootrecipe']
 
 depends:
-    # devel stuff (usually) not deployed to targets -> host compat toolchain
-    - devel::byacc
-    - devel::dtc
-    # - devel::intltool # FIXME: configure: error: XML::Parser perl module is required for intltool
-    - devel::lua-bin
-    - devel::makedepend
-    - devel::nasm
-    - devel::rpcsvc-proto
-    - devel::squashfs-tools
+    - use: [tools]
```

Comment:
I don't think these tools will be built then. For example, I cannot find any use of the "byacc" tool. I guess you want to use `use: [result]` instead. This will build the dependency but ignore provided dependencies.

Reply:
right....  :facepalm: